.gitmodules

```diff
@@ -7,3 +7,6 @@
 [submodule "lib/raygui"]
 	path = lib/raygui
 	url = https://github.com/raysan5/raygui.git
+[submodule "tracy"]
+	path = lib/tracy
+	url = git@github.com:wolfpld/tracy.git
```

Comment:
I pinned this to v8.2 so it needs the corresponding server executable. https://github.com/wolfpld/tracy/releases/tag/v0.8.2